tests/app_9/CMakeLists.txt

```diff
@@ -1,14 +1,20 @@
+cmake_minimum_required(VERSION 3.0)
+project (test D)
 include(UseDUnittest)
 
+set(CMAKE_BUILD_TYPE DEBUG)
```

Comment:
`tests/run_tests_linux.sh` tests all the build types. Don't fix it here, just use the script (cmake_minimum_required(VERSION 3.0); project (test D); enable_testing() are also not needed)